task/orka-full/0.1/README.md

```diff
@@ -122,7 +91,17 @@ stringData:
   password: admin
 ```
 
-#### Using an SSH key
+> **TIP:** Did you know you could use a script for easier setup?
+>
+> curl -LO https://raw.githubusercontent.com/tektoncd/catalog/master/task/orka-full/0.1/add-orka-creds.sh && chmod 755 add-orka-creds.sh
+> NAMESPACE=\<namespace\> EMAIL=\<email\> PASSWORD=\<password\> ./add-orka-creds.sh --apply
```

Comment:
The formatting of this looks a bit off to me when rendered. Maybe we consider putting these commands inside of a script block, i.e. with triple backticks, or just having the link to SCRIPTS.md

Reply:
Script blocks don't work within the block quote element. Simple code formatting doesn't look good either. Let's just drop these and link to the TL;DR sections instead.